packages/ui-react/src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -19,10 +19,8 @@ const ConfirmationDialog: React.FC<Props> = ({ open, title, body, onConfirm, onC
   const { t } = useTranslation('common');
 
   return (
-    <Dialog open={open} onClose={onClose} role="alertdialog" aria-describedby="confirmation-body" aria-labelledby="confirmation-heading">
-      <h2 className={styles.title} id="confirmation-heading">
-        {title}
-      </h2>
+    <Dialog open={open} onClose={onClose} role="alertdialog" aria-describedby="confirmation-body">
```

Comment:
Does this `aria-describedby` add anything?

Reply:
This was already in the Dialog, but I'll remove it because it doesn't add anything due to `role="alertdialog"`, so the confirmation-body gets read either way. Thanks!